### DISCUSSION TITLE
modern flyouts on windows 8.1

### DISCUSSION THREAD

**Discussion by NfsFanboy03** [ShankarBUS/ModernFlyouts#579] [Ideas]
hi! I have this beautiful app installed on my main pc, i also have a laptop (thinkpad x250) for college, i was wondering if there's any way of making modern flyouts to work with windows 8.1? on my laptop i downgraded back to windows 8.1, as touchscreen gestures are miles better and it runs smoother on windows 8.1. 
The flyouts on windows 8.1 are identical to windows 10, so if it works on windows 10, why cant it on 8.1 😛
thanks!

**Comment by Samuel12321** [comment]
We used to support windows 8 however since 0.4 we only support Windows 10 1809 and above due to lack of support for .msix installer. Additionally npsmlib isn't compatible with Windows 8.

Try if you want to use ModernFlyouts on Windows 8 try version 0.3, it should work but it will be missing bug fixes and features from later releases.

**Reply by NfsFanboy03** [reply]
Thankyou kindly :)

**Comment by Samuel12321** [comment]
Just scroll down further, all releases are on github.
